packages/web3torrent/src/utils/checkTorrentInTracker.ts

```diff
@@ -1,33 +1,29 @@
 import {Client} from 'bittorrent-tracker';
 import {defaultTrackerOpts} from '../constants';
 import {TorrentTestResult} from '../library/types';
-import debug from 'debug';
-const log = debug('web3torrent:tracker-check');
+import {logger} from '../logger';
+const log = logger.child({module: 'tracker-check'});
 
 export async function checkTorrentInTracker(infoHash: string) {
```

Comment:
Should we add a return type to this function?
```suggestion
export async function checkTorrentInTracker(infoHash: string): Promise<TorrentTestResult> {
```

Reply:
https://github.com/statechannels/monorepo/pull/1490/commits/3952c25531b950f53d5f9031024f668171ad4b2c